RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
> Copy-pasted old implementation just can not stay.

Then make a PR and fix it.

You suggested to let the old function throw an exception, and I offered to make a PR for that.